src/index/base.h

```diff
@@ -125,6 +128,8 @@ class BaseIndex : public CValidationInterface
 
     /// Get a summary of the index and its state.
     IndexSummary GetSummary() const;
+
+    void SetBestBlockIndex(const CBlockIndex* block);
```

Comment:
3b8b898 it looks like `SetBestBlockIndex` can be private

Reply:
Right, done.